bin_interface.c

```diff
@@ -143,6 +142,41 @@ int bin_push_int(int info)
 	return (int)(cpos - send_buffer);
 }
 
+/*
+ * removes @count intergers from the end of the send buffer
+ *
+ * @return:
+ *		0: success
+ *		< 0: error, no more integers in buffer
+ */
+int bin_alter_pop_int(int count) {
```

Comment:
could you please rename this function? Looking at the comment and at the name of the function, I don't really understand what it is supposed to do. And since it is in the bin interface API/headers, should be pretty clear what the function should do.